tests/testthat/test-utils.R

```diff
@@ -29,3 +29,11 @@ testthat::test_that("example_file works", {
 
 })
 
+testthat::test_that("dot_to_package", {
+  p0 <- get_root_path("jsonld")
```

Comment:
Could you use a package from Imports as example? I wonder whether using a package from Suggests is something we might regret later.